src/functions/getDefects.js

```diff
@@ -1,20 +1,24 @@
 'use strict'
 
-const getDefectList = require('../services/defectsService')
+const DefectsDto = require('../models/DefectsDto')
```

Comment:
any specific reason we have followed a different naming convention here? everywhere else its following camelCase except this and the following line?